ONSdigital/rm-collection-exercise-service
# Add creation of action plans to create collection exercise

# Motivation and Context
All collection exercises require actionplans to be associated with them. This PR adds these action plans when the exercise is created

# What has changed
Call action service to 2 new action plans for each collection exercise at creation
Update casetypeoverride table to use a sequence for it's primary key
Use new action plans to update the casetypeoverride table

# How to test?
Create collection exercise (through UI or API) and check that new action plans are created and associated with the new exercise
Acceptance tests should still pass

# Links
[Trello](https://trello.com/c/4rHB8eG3/65-add-creation-of-action-plans-to-create-collection-exercise)

# Assoiciated PR's
[ras-rm-docker-dev](https://github.com/ONSdigital/ras-rm-docker-dev/pull/57)
